Jenkinsfile

```diff
@@ -0,0 +1,79 @@
+readTrusted 'Dockerfile_jenkins_ubuntu32'
+
+pipeline {
+  agent none
+
+  stages {
+    stage('64bit') {
+      agent {
+        docker {
+          image 'golang:1.9.2'
+        }
+      }
+
+      steps {
+        sh """
+        mkdir -p /go/src/github.com/influxdata
+        cp -a $WORKSPACE /go/src/github.com/influxdata/influxdb
+
+        cd /go/src/github.com/influxdata/influxdb
+        go get github.com/sparrc/gdm
+        gdm restore
+        """
+
+        sh """
+        cd /go/src/github.com/influxdata/influxdb
+        go test -parallel=1 ./...
```

Comment:
What do you think about archiving the output of `go test`, and running that output through go-junit-report so Jenkins can parse the individual test states?

Reply:
I'm concerned that there's too much output and part of the reason why we never figure out the problems with Circle is because we were always drowned in so much output. I've got an idea for filtering the verbose output. Maybe we can have it so whatever filter will do, it can do junit and also output the full verbose text to a file we can archive?

Reply:
As-is, all of the `go test` output from each stage will be shown in Jenkins' console output page, which is approximately as bad as the current situation with Circle. 

Changing this to `go test -parallel=1 ./... 2>&1 | tee /path/to/full-test-log.txt` would allow us to then archive that file, so we could view the full test output from a single run, and we can convert it to junit so we get the easily navigated individual test breakdown. That's a pretty small change that I would prefer to see now, to hold over until the more sophisticated solution you're describing.